### DISCUSSION TITLE
Having CORS problem with simple fetch

### DISCUSSION THREAD

**Discussion by ludersGabriel** [vercel/next.js#44276] [Help]
Hi! I'm trying to call an ip address API with the following function:

```
const getIp = async (): Promise<string | null> => {
  try {
    const { data } = await axios.get(
      'https://api.ipify.org?format=json'
    )
    return data.ip ?? null
  } catch (err) {
    return null
  }
}

```

But it just wont resolve. CORS crossed origin problem everytime. Tried passing {withCredentials: false} to axios and it did not work as well. Also tried using the fetch api and it didnt work.

**Comment by GabenGar** [comment]
In what context are you trying to call?

**Comment by vitalijalbu** [comment]
https://github.com/vercel/next.js/discussions/45239
Anyone can help guys? Thanks
